blog/wdio-blog.md

```diff
@@ -0,0 +1,284 @@
+# Meta:
+- Title: React Native Application UI testing with WDIO
+
+- Keywords: React Native, iOS and Android UI testing, WebDriverIO, Appium, SauceLabs, Real Device testing
+
+- Summary: We at Godaddy, recently adopted WebDriverIO based UI testing for our React Native application thats in development phase. Benefits of using WDIO included that we could write UI tests similar to how we write tests for the web. Synchronous nature of WebDriverIO helps in writing clean UI tests along with PageObject models. WDIO config allows to plugin SauceLabs Emulators or Real Devices for cloud-based testing.
```

Comment:
Nit: no comma

Maybe even remove GoDaddy entirely. The reader knows that the post comes from GoDaddy.